satpy/tests/test_readers.py

```diff
@@ -447,6 +447,19 @@ def _ignore_all_tags(self, tag_suffix, node):
                              "Reader YAML filename doesn't match reader "
                              "name in the YAML file.")
 
+    def test_available_readers(self):
+        """Test the 'available_readers' function."""
+        from satpy import available_readers
+        reader_names = available_readers()
+        self.assertGreater(len(reader_names), 0)
+        self.assertIsInstance(reader_names[0], str)
```

Comment:
Could we also check that some stable readers are on the list? like viirs, abi...